src/safe-upgrade-tool/run.md

```diff
@@ -0,0 +1,137 @@
+---
+group: software-update-guide
+title: Run the tool
+functional_areas:
+  - Upgrade
+---
+
+The Magento Safe Upgrade Tool (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed on it.
+
+potential problems that must be fixed in your custom code before attempting to upgrade to a newer version of Magento.
+
+The tool returns a list of errors and warnings that you must address before upgrading to a new version of Magento.
+
+## Use SUT
+
+### Memory limitations
+
+The SUT uses memory on your Magento instance to run—even if you installed the SUT on a different machine. If you are running a system with large third-party modules and files, the SUT requires a high amount of RAM.
```

Comment:
Please confirm with the team that this statement is true, sounds strange to me:
`The SUT uses memory on your Magento instance to run - even if you installed the SUT on a different machine`

Reply:
@svera could you help clarify this section?

Reply:
@danidelcar I would get rid of the whole phrase, it doesn't make sense.

Reply:
The whole memory limitations section @svera ?, or just the phrase Alex mentioned earlier:

`The SUT uses memory on your Magento instance to run — even if you installed the SUT on a different machine. `

We do need to highlight the memory limitations we face with the tool I think

Reply:
@del just the phrase Alex mentioned, the memory requirements are important.

Reply:
corrected section, please review @svera & @paliarush  thanks!